hdl/ghdl-prefix.mk

```diff
@@ -0,0 +1,5 @@
+# When using fomu toolchain to synthezise VHDL, GHDL_PREFIX has to be set
+# See https://github.com/YosysHQ/fpga-toolchain#using-ghdl
+
+GHDL_PREFIX ?= $(shell dirname $(shell ghdl --libghdl-library-path))/ghdl
+export GHDL_PREFIX := $(GHDL_PREFIX)
```

Comment:
I guess that https://github.com/im-tomu/fomu-workshop/blob/master/.github/tests.sh#L9 can be removed, as it is redundant.